persistence/src/test/resources/db/test-data/R__insert_values.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO roll.project (owner_id, name, description, status) VALUES (gen_random_uuid(), 'Project', 'description', 'active');
```

Comment:
to build better(transparent) test data you need to explicitly set the id  `setval('roll.project_id_seq', 1000)`
Then in actual test `projectRepository.findById(1000L)` looks and works much predictable